lib/import/publishing_api_importer.rb

```diff
@@ -0,0 +1,100 @@
+module Import
+  class PublishingApiImporter
+
+    def initialize(checker_db, progress_reporter, publishing_api_url)
+      @checker_db = checker_db
+      @thread_pool = Thread.pool(1)
```

Comment:
The pools in each importer are for additional work that needs to be performed when an api response arrives.
The main api iteration for each importer happens on separate threads owned by the CheckRunner. The two importers have different work to perform when a response comes in - in particular, the Rummager importer actually calls out to publishing api. As a result fewer auxiliary threads are needed by the Publishing API importer.
This could all be much more elegant but we've reached (and passed) the point of diminishing returns IMO.
Also note that the pools are sized for a dev VM - presumably we could hammer the apis harder from Jenkins in integration but there's not much need since the import takes ~15mins.